components/mpas-albany-landice/cime_config/testdefs/testmods_dirs/mali/gis/user_nl_mali

```diff
@@ -0,0 +1,2 @@
+config_am_globalstats_enable = .false.
+config_am_regionalstats_enable = .false.
```

Comment:
Do you mean to rename the `gis20km` testmod to `gis` but also add a testmod called `landiceIG`?  It looks like the `gis` is not used in any tests (and is largely redundant).  Can it be deleted?

Reply:
I meant to remove the 'gis*' subdir altogether since it is redundant w/ the new 'landiceIG' subdir now. I'll see what happened there and remedy that. As we discussed previously, the 'landiceIG' name is just a placeholder for the time being, e.g. if we decide that should be changed during this discussion.

Reply:
This has been fixed (redundant 'gis' dir has been removed).